src/components/Footer.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import { ReactComponent as Home } from '../images/home.svg'
```

Comment:
You could also think about grouping your icons in one Icons.js file and export them from there. Makes the import in the final files shorter :)